build.assets/robotest_run_suite.sh

```diff
@@ -22,15 +22,23 @@ readonly ROBOTEST_SCRIPT=$(mktemp -d)/runsuite.sh
 
 # number of environment variables are expected to be set
 # see https://github.com/gravitational/robotest/blob/master/suite/README.md
-export ROBOTEST_VERSION=${ROBOTEST_VERSION:-stable-gce}
+export ROBOTEST_VERSION=${ROBOTEST_VERSION:-uid-gid}
```

Comment:
`uid-gid` vs semver

Reply:
I agree.  I've got a bit more work to do on the Robotest side before I cut back to semver.  This is tracked in https://github.com/gravitational/robotest/issues/200